db/migrate/20251016013505_convert_watched_precedence_over_muted_to_default_pref.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+class ConvertWatchedPrecedenceOverMutedToDefaultPref < ActiveRecord::Migration[8.0]
+  def up
+    execute <<~SQL
+      UPDATE site_settings SET name = 'default_watched_precedence_over_muted'
+      WHERE name = 'watched_precedence_over_muted';
+    SQL
```

Comment:
If we rename this in a pre-deploy migration, it will remove the override and calls to `SiteSetting. watched_precedence_over_muted` will start returning the default instead of the override that the admin has set. It is abit annoying to do but I believe we need to deprecate this site setting first.

Reply:
Yea I think insert then delete in a post deploy migration works too as the odds of an admin updating a site setting while a deploy is happening is low.

> We have also been doing this UPDATE method a bunch of times historically, some even this year, without any known issue so I don't think it's too risky.

I do think there is a bit of survivorship bias here as we don't actually know if things went wrong when we took this approach.